src/js/menu/menu-button.js

```diff
@@ -58,7 +59,7 @@ class MenuButton extends ClickableComponent {
     this.buttonPressed_ = false;
     this.el_.setAttribute('aria-expanded', 'false');
 
-    if (this.items && this.items.length === 0) {
+    if (this.items && (this.items.length === 0 || (this.items.length === 1 && this.items[0] instanceof OffTextTrackMenuItem))) {
```

Comment:
Oh, my mistake - I was thinking of the `CaptionsSettings` menu item, not the `OffTextTrackMenuItem`.